app/renderer/components/Session/CloudProviderSelector.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import _ from 'lodash';
+import { Modal, Row, Col, Button } from 'antd';
+import CloudProviders from './CloudProviders';
+import SessionStyles from './Session.css';
+
+export default class CloudProviderSelector extends Component {
+
+  constructor (props) {
+    super(props);
+    this.state = {
```

Comment:
I think there is a shortcut to this, so you could simply declare state without declaring the constructor.